scripts/lint-style.lean

```diff
@@ -125,8 +134,10 @@ def lintStyleCli (args : Cli.Parsed) : IO UInt32 := do
   -- This script is re-run each time, hence is immune to such issues.
   let nolints ← IO.FS.lines ("scripts" / "nolints-style.txt")
```

Comment:
I've tested in a local project with mathlib set to the latest commit here and get

```
jon@computer TestNormal % lake exe lint-style TestNormal
uncaught exception: no such file or directory (error code: 2)
  file: scripts/nolints-style.txt
```

Reply:
Thanks for testing! I've pushed a fix.

Reply:
Would you like to re-test?